docs/reference/technology/connectivity/connectivity.md

```diff
@@ -101,6 +102,12 @@ Near-field communication (NFC) is a short range (few centimeters) wireless techn
 
 To learn how to use NFC with Mbed OS, please refer to the [Mbed OS NFC overview](../apis/nfc.html).
 
+#### NB-IoT Cellular
+
+Non-IP Data Delivery(NIDD) is a new feature for communication over NB-IoT. It is enabled by Control Plane CIoT EPS optimization and meant to provide improved support of small data transfer. It does this by transporting user data encapsulated in NAS messages, thus reducing the total number of control plane messages when handling a short data transaction.
+
```

Comment:
"user data over the control channel", no need to mention the NAS here.

Reply:
done